met-web/src/components/engagement/old-view/widgets/Video/VideoWidgetView.tsx

```diff
@@ -69,35 +142,126 @@ const VideoWidgetView = ({ widget }: VideoWidgetProps) => {
+                        }}
+                        url={videoWidget.video_url}
+                        controls
+                        onReady={(player) => {
+                            const videoSource = getVideoSource(videoWidget.video_url);
+                            setVideoOverlayTitle(getVideoTitle(player, videoSource, videoWidget.title));
+                        }}
+                        onPlay={() => setShowOverlay(false)}
+                        width="100%"
+                        height={'100%'}
+                        config={playerConfig}
+                        style={{
+                            position: 'absolute',
+                            top: 0,
+                            left: 0,
+                            width: '100%',
+                            height: '100%',
+                        }}
+                    />
+                </AspectRatioContainer>
+            </Grid>
+            <Grid item xs={12} sx={{ pt: '0.5rem !important' }}>
+                <BodyText
+                    style={{ fontSize: '0.95rem', color: isDarkMode ? colors.surface.white : Palette.text.primary }}
+                >
+                    {videoDuration}
+                </BodyText>
             </Grid>
-        </MetPaper>
+        </Grid>
+    );
+};
+
+interface VideoOverlayProps {
+    videoOverlayTitle: string;
+    source: string;
+}
+
+const VideoOverlay = ({ videoOverlayTitle, source }: VideoOverlayProps) => {
+    const getIcon = (source: string) => {
+        if ('YouTube' === source) {
+            return faYoutube;
+        } else if ('Vimeo' === source) {
+            return faVimeo;
+        } else if ('Facebook' === source) {
+            return faFacebookF;
+        } else if ('Twitch' === source) {
+            return faTwitch;
+        } else if ('SoundCloud' === source) {
+            return faSoundcloud;
+        } else if ('Mixcloud' === source) {
+            return faMixcloud;
+        } else if ('DailyMotion' === source) {
+            return faDailymotion;
+        } else {
+            return faQuestionCircle;
+        }
+    };
+    return (
+        <Box
+            sx={{
+                display: 'flex',
+                height: '3.8rem',
+                background: 'black',
+                width: '100%',
+                borderRadius: '16px 16px 0 0',
+                position: 'relative',
+                top: '0',
+                zIndex: '10000',
+                alignItems: 'center',
+                padding: '1rem',
+            }}
+        >
+            <div
+                style={{
+                    alignItems: 'center',
+                    display: 'flex',
+                }}
+            >
+                <FontAwesomeIcon
+                    icon={getIcon(source)}
+                    style={{ fontSize: '1.9rem', paddingRight: '0.65rem', color: '#FCBA19' }}
+                />{' '}
+                <span style={{ fontSize: '0.95rem' }}>{videoOverlayTitle}</span>
```

Comment:
Please stop creating text with size `0.95rem`. This does not correspond to any value used in the designs, and causes rendering issues because it comes out to a fractional pixel value. 
to match the designs in figma, use one of the following:
1) <BodyText size="small">
2) fontSize: '0.875rem'
3) fontSize: '14px'